Automattic/jetpack
# cli: Add timing to build output

#### Changes proposed in this Pull Request:

`jetpack build --timing` will now output timing information for each
completed command. With `--verbose`, it will also prefix each line of
output with the time since the start of the task.

#### Jetpack product discussion


PT: p9dueE-4dJ-p2

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Try `jetpack build`'s new `--timing` option. Does it work correctly in various use cases?
* Check the Build job logs, does the timing information show up properly there?
